### DISCUSSION TITLE
Project 'react-native-reanimated' not found in root project

### DISCUSSION THREAD

**Discussion by akbari6776** [software-mansion/react-native-reanimated#3503] [Q&A]
In my react native project I don't have `"react-native-reanimated"` but when I rebuilt my project I got this error:
`"Project "react-native-reanimated" was not found in the root project`

So when I install ‍`react-native-reanimated‍` I get an `NDK` install error, but when I install the `ndk` and get the `apk` from the project, the app won't open and crashes.

So I uninstalled `react-native-reanimated` and `ndk` and `cmake` but again I got the error `"project 'react-native-reanimated' was not found in the root project"`

please help me

<img width="716" alt="Screen Shot 2022-08-20 at 15 26 58" src="https://user-images.githubusercontent.com/42941890/185742865-8b5ffbbb-7b57-44b3-aff6-69965b70f8b2.png">

**Comment by ayozebarrera** [comment]
Try deleting the .gradle folder in your project's root folder! Sometimes it's get cached

Don't forget to resync your project from Android Studio
